packages/manager/modules/billing/src/orders/billing-orders-main.controller.js

```diff
@@ -7,27 +7,27 @@ export default class BillingOrdersMainCtrl {
   /* @ngInject */
   constructor(
     $translate,
-    constants,
     currentUser,
     featuresAvailabilities,
     purchasesOrdersSectionState,
     ordersSectionState,
+    guides,
+    trackClick,
   ) {
     this.$translate = $translate;
     this.featuresAvailabilities = featuresAvailabilities;
-    this.constants = constants;
     this.currentUser = currentUser;
     this.purchasesOrdersSectionState = purchasesOrdersSectionState;
     this.ordersSectionState = ordersSectionState;
+    this.guides = guides;
+    this.trackClick = trackClick;
+    console.log(this.guides);
```

Comment:
**nitpick:** Could you please remove this console statement?

Reply:
Oops, I forgot. It's fixed